vars/secretScanner.groovy

```diff
@@ -1,9 +1,9 @@
 import uk.gov.defra.ffc.SecretScanner
 
-def scanWithinWindow(githubCredentialId, dockerImgName, githubOwner, repositoryPrefix, scanWindowHrs, excludeStrings, slackChannel="") {
+def scanWithinWindow(String githubCredentialId, String dockerImgName, String githubOwner, String repositoryPrefix, String scanWindowHrs, String excludeStrings, String slackChannel="") {
```

Comment:
I _think_ `scanWindowHrs` should be `Date`.

I also think `excludeStrings` should be an array.  ie `String[]`